src/test/kotlin/com/igorwojda/integer/power/challenge.kt

```diff
@@ -2,9 +2,15 @@ package com.igorwojda.integer.power
 
 import org.amshove.kluent.shouldBeEqualTo
 import org.junit.jupiter.api.Test
+//
+//private fun power(base: Int, exponent: Int): Int {
+//    if (exponent > 0)
+//        return base * power(base, exponent-1)
+//    return 1
+//}
 
 private fun power(base: Int, exponent: Int): Int {
-    TODO("not implemented")
+    return List(exponent) { base }.reduce{multi, elem -> multi * elem}
```

Comment:
Very good! But, you're still missing point 3 of the challenge: A method reference (to the above operator): https://khan.github.io/kotlin-for-python-developers/#function-references